packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```
@@ -22,30 +22,32 @@
 
 .bpk-chip-group-container {
   display: flex;
+  margin: 0 (-4 * tokens.$bpk-one-pixel-rem);
   align-items: center;
   white-space: nowrap;
 }
 
 .bpk-chip-group {
   display: flex;
   margin: 0;
-  padding: 0;
+  padding: 4 * tokens.$bpk-one-pixel-rem;
```

Can we use `bpk-spacing-sm()`which is 4px

```suggestion
  padding: tokens.bpk-spacing-sm();
```